### PR TITLE
Merge Outfits (and related products update) into Main

### PR DESCRIPTION
**Pull request by lawlorseanr**

### CODE REVIEW

File: client/src/components/RelatedItems/Compare/CompareAction.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import actions from '../../../state/actions/index.js';
+
+export default ({ product }) => {
+  const dispatch = useDispatch();
+  return (
+    <div
+      className='related-products-image-modalopen'
+      onClick={e => {
+        dispatch(actions.setViews.modalView(product.productInfo));
+        e.stopPropagation();
```
Comment (tyler-audio) [inline client/src/components/RelatedItems/Compare/CompareAction.jsx:13]: Is this line necessary?

File: client/src/components/RelatedItems/Compare/CompareModal.jsx [hunk 2]
```diff
@@ -13,27 +13,26 @@ export default () => {
...
+      className='related-products-modal-overlay'>
+      <div className='related-products-modal-background'>
         <span
-          style={{cursor: 'pointer', float: 'right', padding: '5px 5px 0 0'}}
+          id='modal-close-button'
           onClick={e => {
             dispatch(actions.setViews.modalView());
             e.stopPropagation();
           }}>
           {'X'}
```
Comment (tyler-audio) [inline client/src/components/RelatedItems/Compare/CompareModal.jsx:28]: I found that a better method for creating an 'x' close button is to use the html entity &times;
The &times; entity is easier to center and whatnot, while the letter 'x' will scale differently based on it's font size, and is generally more annoying to work with.

File: client/src/components/RelatedItems/Outfit/OutfitAction.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import actions from '../../../state/actions/index.js';
+
+export default ({ product }) => {
+  const dispatch = useDispatch();
+  return (
+    <div
+      className='related-products-image-outfitremove'
+      onClick={e => {
+        dispatch(actions.selectOutfits.removeOutfit(product));
+        console.log('remove!');
```
Comment (tyler-audio) [inline client/src/components/RelatedItems/Outfit/OutfitAction.jsx:13]: remove that pesky console log

File: client/src/components/RelatedItems/OutfitList.jsx [hunk 4]
```diff
@@ -30,14 +34,61 @@ const OutfitButton = ({ direction }) => {
...
 
+/* =============================================
+             Outfits
+============================================= */
+const Outfits = outfits => {
+  return _.map(outfits, (outfit, i) =>
+    <div key={i} className='related-products-card-container'>
+      <ProductCard
+        product={outfit}
+        action={OUTFIT_ACTION}/>
```
Comment (tyler-audio) [inline client/src/components/RelatedItems/OutfitList.jsx:45]: What is the reason for using OUTFIT_ACTION here instead of just 'remove'?

**Review by tyler-audio — APPROVED** [review]

File: client/src/components/RelatedItems/Compare/CompareModal.jsx (hunk 2 above)
Comment (lawlorseanr) [reply client/src/components/RelatedItems/Compare/CompareModal.jsx:28]: i would love to just refactor my entire modal implementation to follow what you did. i added it to the backlog :)

File: client/src/components/RelatedItems/Compare/CompareAction.jsx (hunk 1 above)
Comment (lawlorseanr) [reply client/src/components/RelatedItems/Compare/CompareAction.jsx:13]: If you mean the 'e.stopPropagation()' then yes. This prevents the components below (z-index) from activating their onClick events.

File: client/src/components/RelatedItems/Outfit/OutfitAction.jsx (hunk 3 above)
Comment (lawlorseanr) [reply client/src/components/RelatedItems/Outfit/OutfitAction.jsx]: removed!

**Pull request merged**